src/deprecated.jl

```diff
@@ -1 +1,5 @@
-@Base.deprecate predict(mm::LinearModel, newx::AbstractMatrix, interval::Symbol, level::Real = 0.95) predict(mm, newx; interval=interval, level=level)
\ No newline at end of file
+using Base: deprecate
```

Comment:
Doesn't seem to be used.

Reply:
Apparently that's equivalent to `@deprecate`, but let's change that.

Reply:
Is it? I think this worked before because `@deprecate` is exported from Base.

Reply:
Woops, you're right. :-D